### PR TITLE
Warn user if tipping `http` URL

### LINKED ISSUE #770: Warn user if they are tipping `http`
ref https://github.com/aeternity/superhero-ui/issues/953

### PR DESCRIPTION
**Pull request by imollov**
closes #770

Screenshot 2021-01-20 at 13 53 46

Screenshot 2021-01-20 at 13 48 37

@aniputapet please advise what should be the title and description of the modal window above.

### CODE REVIEW

File: src/store/plugins/tipUrl.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ export default (store) =>
         let status;
         if (blacklistedUrls.some((u) => url.includes(u))) {
           status = 'blacklisted';
+        } else if (url.startsWith('http') && !url.startsWith('https:')) {
```
Comment (davidyuk) [inline src/store/plugins/tipUrl.js:20]: What if I'm donating to http.com? Also, let's normalize URL before validation (https://github.com/aeternity/superhero-ui/issues/996)

**Review by mradkov — APPROVED** [review]
Code changes looking good. @aniputapet can you provide text for the exact warning message?

**Review by davidyuk — APPROVED** [review]

**Comment by aniputapet** [comment]
Superhero Wallet is marking HTTP sites as "Not secure” - please, use URLs with HTTPS, only. 

The S in HTTPS stands for "secure." The TLS (or SSL) certificate proves that the server is actually the legitimate host of the website.  Website using HTTPS are more trustworthy and secure, for both users and website owners. 


@mradkov

**Comment by mradkov** [comment]
> Superhero Wallet is marking HTTP sites as "Not secure” - please, use URLs with HTTPS, only.
> 
> The S in HTTPS stands for "secure." The TLS (or SSL) certificate proves that the server is actually the legitimate host of the website. Website using HTTPS are more trustworthy and secure, for both users and website owners.
> 
> @mradkov

@imollov please add the following and rebase this PR so we can merge it. ✌️

**Review by mradkov — APPROVED** [review]

**Pull request merged**